src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -692,7 +692,7 @@ private function addServices()
         return $publicServices.$privateServices;
     }
 
-    private function addNewInstance(Definition $definition, $return, $instantiation)
+    private function addNewInstance(Definition $definition, $return, $instantiation, $id)
```

Comment:
i realized that the id parameter was removed since symfony 2 (i tested this in my project with symfony 2...)

not sure if its ok to pass the id around just for the purpose of the error message, but without the id the message becomes a lot less useful